### PR TITLE
Creation flow

### PR DESCRIPTION
**Pull request by SiimonStark**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

`When Navlink "Create New Project" is clicked it now routes you to "/createNewProject"`

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`I also have it set to where the "save" button changes based on whether it's new or not`

## Has This Been Tested?

- [X] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`Your message here`

# Message for reviewer:
`Your message here`

# Checklist:

- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

Co-authored-by: Erik Konner <erikkonner@gmail.com>
Co-authored-by: Justin Pyktel <justinpyktel@gmail.com>

### CODE REVIEW

File: src/components/app/App.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ export class App extends Component {
       <Header />
       <main>
         <Route exact path="/" component={ControlForm} />
+        <Route exact path="/createNewProject" component={ControlForm} />
```
Comment (ehk9000) [inline src/components/app/App.js]: sweet fix!

File: src/components/control_form/index.js [hunk 2]
```diff
@@ -4,27 +4,48 @@ import { connect } from 'react-redux';
...
     this.state = {
-      project_name: '',
+      project_name: this.setName(),
       project_palettes: []
     }
   }
 
   componentDidMount() {
-    let { palettes, currentProject } = this.props
+    // let { palettes, currentProject } = this.props;
```
Comment (ehk9000) [inline src/components/control_form/index.js]: Do we have a plan for this componentDidMount? do we need it anymore?

File: src/components/control_form/index.js [hunk 3]
```diff
@@ -58,13 +79,24 @@ export class ControlForm extends Component {
...
+  toggleProjectSave = () => {
+    let value;
+
+    this.props.currentProject.name
+      ? value = 'Update'
+      :value = 'Save';
+    
+      return value;
+  }
+
```
Comment (ehk9000) [inline src/components/control_form/index.js]: right on !

File: src/components/control_form/index.js [hunk 4]
```diff
@@ -58,13 +79,24 @@ export class ControlForm extends Component {
...
   render() {
+    // let project = this.loadProject();
```
Comment (ehk9000) [inline src/components/control_form/index.js]: do we need this log?

File: src/components/control_form/index.js (hunk 2 above)
Comment (SiimonStark) [reply src/components/control_form/index.js]: I fixed that on the [Rebase PR](https://github.com/ehk9000/Palette-Picker-UI/pull/50)

File: src/components/control_form/index.js (hunk 4 above)
Comment (SiimonStark) [reply src/components/control_form/index.js]: I don't remember if I removed this. Will on next PR if I didn't

**Pull request merged**